### PR TITLE
Support for iOS 14 Xcode 12

### PR DESCRIPTION
**Pull request by IlyaBausovAkvelon**
1. Updated Frameworks
2. Fixed arm64 architecture issue for iOS 14 simulators.

### CODE REVIEW

**Review by AndreyMaslennikov — APPROVED** [review]

File: Frameworks/FBControlCore.framework/Versions/A/Headers/FBControlCoreConfigurationVariants.h [hunk 1]
```diff
@@ -112,10 +122,12 @@ extern FBOSVersionName const FBOSVersionNameiOS_13_1;
 extern FBOSVersionName const FBOSVersionNameiOS_13_1_1;
 extern FBOSVersionName const FBOSVersionNameiOS_13_1_2;
 extern FBOSVersionName const FBOSVersionNameiOS_13_1_3;
+extern FBOSVersionName const FBOSVersionNameiOS_13_2;
 extern FBOSVersionName const FBOSVersionNameiOS_13_2_1;
 extern FBOSVersionName const FBOSVersionNameiOS_13_3;
 extern FBOSVersionName const FBOSVersionNameiOS_13_3_1;
 extern FBOSVersionName const FBOSVersionNameiOS_13_4;
+extern FBOSVersionName const FBOSVersionNameiOS_13_5;
 extern FBOSVersionName const FBOSVersionNametvOS_9_0;
```
Comment (ivan-nosar) [inline Frameworks/FBControlCore.framework/Versions/A/Headers/FBControlCoreConfigurationVariants.h:141]: Where is iOS 14?

File: Frameworks/FBControlCore.framework/Versions/A/Headers/FBControlCoreConfigurationVariants.h [hunk 2]
```diff
@@ -77,6 +84,8 @@ extern FBDeviceModel const FBDeviceModelAppleWatchSeries3_38mm;
 extern FBDeviceModel const FBDeviceModelAppleWatchSeries3_42mm;
 extern FBDeviceModel const FBDeviceModelAppleWatchSeries4_40mm;
 extern FBDeviceModel const FBDeviceModelAppleWatchSeries4_44mm;
+extern FBDeviceModel const FBDeviceModelAppleWatchSeries5_40mm;
+extern FBDeviceModel const FBDeviceModelAppleWatchSeries5_44mm;
```
Comment (ivan-nosar) [inline Frameworks/FBControlCore.framework/Versions/A/Headers/FBControlCoreConfigurationVariants.h:92]: Where is Watch 6?

File: Frameworks/FBControlCore.framework/Versions/A/Headers/FBControlCoreConfigurationVariants.h (hunk 1 above)
Comment (AndreyMaslennikov) [reply Frameworks/FBControlCore.framework/Versions/A/Headers/FBControlCoreConfigurationVariants.h:141]: Added

File: Frameworks/FBControlCore.framework/Versions/A/Headers/FBControlCoreConfigurationVariants.h (hunk 2 above)
Comment (AndreyMaslennikov) [reply Frameworks/FBControlCore.framework/Versions/A/Headers/FBControlCoreConfigurationVariants.h:92]: Added

**Review by ivan-nosar — APPROVED** [review]
The CI checks seem to be failed on the last commit. Please make sure about fixing it before making the merge
cc @AndreyMaslennikov @IlyaBausovAkvelon

**Pull request merged**